### DISCUSSION TITLE
Duda guardar cabezas (leer input código inicial)

### DISCUSSION THREAD

**Discussion by Catalinapadilla** [IIC2133-PUC/2024-1#72] [Tarea 1] [answered]
Hola!
Me pasa que, al intentar guardar las cabezas, me sale error en la primera. No he modificado nada de lo que entregaron e igual borré todos los cambios y utilicé el código tal cual como estaba al principio y, aun así, me sigue dando error. Eso sí, el número de cabezas se lee bien, pero luego, al leer los datos de la primera cabeza, falla y si hago un print del valor de result, me sale que es 0.

**Comment by tomastrivino** [comment, marked as answer]
Agrégale `\n` a los fscanf.
`  result = fscanf(input_file, "%d\n", &N_HEADS);`
`    result = fscanf(input_file, "SAVE %d %s %d %d %d %d\n", &h->id, h->name, &h->year, &h->region, &h->x, &h->y);`

Eso te lo debería arreglar :3

**Comment by sachondo7** [comment]
Efectivamente la solución del compañero de arriba debería ser suficiente, intentalo y si no avisanos para darte otra alternativa.
